src/node/stage/approved.rs

```diff
@@ -59,368 +49,249 @@ const KNOWLEDGE_TIMEOUT: Duration = Duration::from_secs(2);
 /// This helps avoid relocated node receiving message they need to process from previous section.
 const INITIAL_RELOCATE_COOL_DOWN_COUNT_DOWN: i32 = 10;
 
-struct CompleteParsecReset {
-    /// The new genesis prefix info.
-    pub gen_pfx_info: GenesisPfxInfo,
-    /// The cached events that should be revoted. Not shared state: only the ones we voted for.
-    /// Also contains our votes that never reached consensus.
-    pub to_vote_again: BTreeSet<NetworkEvent>,
-    /// The events to process. Not shared state: only the ones we voted for.
-    /// Also contains our votes that never reached consensus.
-    pub to_process: BTreeSet<NetworkEvent>,
-    /// Event to send on completion.
-    pub event_to_send: Option<Event>,
-}
-
-#[derive(Debug, PartialEq, Eq, PartialOrd, Ord, Hash, Clone)]
-enum PendingMessageKey {
-    NeighbourInfo {
-        version: u64,
-        prefix: Prefix<XorName>,
-    },
-}
-
-pub struct ElderDetails {
-    pub chain: Chain,
-    pub network_service: NetworkService,
-    pub event_backlog: Vec<Event>,
-    pub full_id: FullId,
-    pub gen_pfx_info: GenesisPfxInfo,
-    pub msg_queue: VecDeque<QueuedMessage>,
-    pub sig_accumulator: SignatureAccumulator,
+// The approved stage - node is a full member of a section and is performing its duties according
+// to its persona (infant, adult or elder).
```

Comment:
`DataManager`, `ClientManager`, etc... are vault-specific terms. Routing doesn't know them. Also this did not change, we use the term persona in other places in routing (e.g. the `MemberPersona` enum). It might not be the most appropriate term and we might want to change it, but it seems to be outside of the scope of this PR.